### DISCUSSION TITLE
Noticed on aliexpress there are some excellently priced esp32-s3 backbox mounting touchscreens

### DISCUSSION THREAD

**Discussion by StuartIanNaylor** [HASwitchPlate/openHASP#804] [Hardware]
https://www.aliexpress.com/item/1005006381120764.html ￡30.79

Its a S3-Wrover-N16R8 so 16MB flash 8MB psram and wondering if anybody has given them a go or done any prebuilt images?

**Comment by fvanroie** [comment]
It seems a rebranded Panlee panel, so that should work already with the correct config.

**Reply by StuartIanNaylor** [reply]
Thnx

**Reply by fvanroie** [reply]
I think it's this config https://github.com/HASwitchPlate/openHASP/blob/master/user_setups/esp32s3/zw3d95ce01s_ar.ini
There is an AR/TR/UR version, so check the device for the correct firmware file.

**Reply by StuartIanNaylor** [reply]
Thnx on the sales pic the pcb says TR but will see what arrives.
PS and apols but with 16MB/8MB flash/psram is the max of 12 pages about optimal or can it do more?
http://doc.panel-tag.com/_static/espboard/ESP32S3/ZX3D95CE01S-UR/ZX3D95CE01S-UR-4848.pdf

**Reply by fvanroie** [reply]
You can add more pages if you compile the firmware yourself... not tested though, but it *should* be fine.
